giza/giza/content/newTables/views.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2015 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from rstcloth.rstcloth import RstCloth
+
+logger = logging.getLogger('giza.content.newTables.views')
+
+
+def render_table(rows):
+    r = RstCloth()
+
+    r.directive(name="list-table", fields=[("header-rows", "1") ])
```

Comment:
there's code in rstcloth that builds tables using python, rather than
this aproach, which is slightly more primitive, and requires you to
do a bit more string handling